markdown/frameworks/framework_einstieg.md

```diff
@@ -0,0 +1,146 @@
+---
+type: lecture-cg
+title: "Frameworks: Einstieg"
+menuTitle: "Frameworks"
+author: "André Matutat (FH Bielefeld)"
+weight: 1
+readings:
+  - key: ""
+tldr: |
+  Frameworks sind ein Softwaregerüst, um wiederkehrende Softwarestrukturen schnell und einfach um die eigene Funktionalität zu erweitern.
+  Im Gegensatz zu Libraries stellen sie keine einzelnen Funktionen bereits, die im eigene Code aufgerufen werden, sondern geben Strukturen vor, in denen der eigenen Code eingearbeitet werden muss.
+outcomes:
+  - k1: "Begriff Framework kennen gelernt"
+  - k2: "Unterschied zwischen Frameworks und Libraries verstehen"
+quizzes:
+  - link: "XYZ"
+    name: "Quiz XXX (ILIAS)"
+youtube:
+  - link: ""
+    name: "VL "
+  - link: ""
+    name: "Demo "
+  - link: ""
+    name: "Demo "
+fhmedia:
+  - link: ""
+    name: "VL "
+sketch: true
+---
+
+
+## Motivation
+
+Neben der eigentlichen Funktionalität werden an Softwareprodukte viele weitere Randbindungen gestellt.
+Dies können Sicherheitsbindungen sein oder Anforderungen an die Art und Weise der Bereitstellung des Produktes, zum Beispiel als Webservice.
+Viele dieser Anforderungen ähneln sich von Produkt zu Produkt und müssen dennoch immer wieder neu programmiert werden.
+
+Für einen Webservice muss eine HTML-Schnittstelle bereitgestellt werden, die Anfragen annimmt, bearbeitet und die Antworten wieder zurückliefert.
+
+## Was sind Frameworks?
```

Comment:
Mit der direkten Übernahme von Texten (aka Zitat) tue ich mich im Umfeld von OER schwer. Hier würde ich lieber einen eigenen Text/Stichpunkte sehen und einen Verweis (Link) in die Wikipedia.